### PR TITLE
V3.0.0 alpha bed mesh in macro corrections

### PR DESCRIPTION
**Pull request by Benoitone**
for optional bed mesh in macros to avoid Unknown command:"BED_MESH_CLEAR"

### CODE REVIEW

**Comment by Frix-x** [comment]
Hello, thanks for this PR.

I'm almost ready to merge it but I think you forgot to include the ERCF_EJECT macro definition that you used. Can you add it to the PR to make it work correctly ?

Thanks you :)

File: macros/base/cancel_print.cfg [hunk 1]
```diff
@@ -8,18 +8,18 @@ gcode:
...
+    {% set bed_mesh_enabled = printer["gcode_macro _USER_VARIABLES"].bed_mesh_enabled %}
 
     PARK
-    
+
     {% if ercf_enabled %}
         # unload filament and park into ercf
-        _ERCF_FORM_TIP_STANDALONE
-        ERCF_UNLOAD
+        ERCF_EJECT
```
Comment (Frix-x) [inline macros/base/cancel_print.cfg:17]: This macro currently doesn't exist currently. Can you add it ?

File: macros/base/end_print.cfg [hunk 2]
```diff
@@ -9,32 +9,36 @@ gcode:
...
     {% set status_leds_enabled = printer["gcode_macro _USER_VARIABLES"].status_leds_enabled %}
+    {% set bed_mesh_enabled = printer["gcode_macro _USER_VARIABLES"].bed_mesh_enabled %}
 
     PARK
 
     {% if ercf_enabled %}
         # unload filament and park into ercf
-        _ERCF_FORM_TIP_STANDALONE
-        ERCF_UNLOAD
+        ERCF_EJECT
```
Comment (Frix-x) [inline macros/base/end_print.cfg:18]: This macro currently doesn't exist currently. Can you add it ?

**Review by Frix-x — CHANGES_REQUESTED** [review]

File: macros/base/end_print.cfg (hunk 2 above)
Comment (Benoitone) [reply macros/base/end_print.cfg:18]: ERCF_EJECT is on the ercf.py script

Comment (Benoitone) [reply macros/base/end_print.cfg:18]: And depreciate in moggieuk scripts.

Comment (Frix-x) [reply macros/base/end_print.cfg:18]: Ohhh thanks. I don't own an ERCF at the moment and was not aware of that. Ok I'll merge :)

**Pull request merged**